src/Xamarin.Android.Build.Tasks/Utilities/ResourceParser.cs

```diff
@@ -48,10 +50,21 @@ internal string GetResourceName (string type, string name, Dictionary<string, st
 				return ResourceIdentifier.CreateValidIdentifier (mappedValue.Substring (mappedValue.LastIndexOf (Path.DirectorySeparatorChar) + 1));
 			}
 
-			Log.LogDebugMessage ("  - Not remapping resource: {0}.{1}", type, name);
-
```

Comment:
I saw this message ~2700 times in a build log. It is still logging the *other* case, which I think is sufficient.